src/components/ScrollToTop.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import { withRouter } from "react-router-dom";
+
+class ScrollToTop extends React.Component {

```

Comment:
As I understand this component scrolls page to the top in the case when the user changes the page. But the page should be scrolled to the top by itself, do you really need this component?

Reply:
I have never seen that after changing the page it doesn't scroll to the top by itself.